develop/schemadoc/Protocol/Protocol.SystemOptions.md

```diff
@@ -0,0 +1,13 @@
+---
+uid: Protocol.SystemOptions
+---
+
+# SystemOptions element
+
+## Parent
+
+[Protocol](xref:Protocol)
+
+## Children
+
+[RunInSeparateInstance](xref:Protocol.SystemOptions.RunInSeparateInstance)
```

Comment:
```suggestion
|Name|Occurrences|Description|
|--- |--- |--- |
|***All***|||
|&nbsp;&nbsp;[RunInSeparateInstance](xref:Protocol.SystemOptions.RunInSeparateInstance)|[0, 1]|Allows you to flag a protocol as requiring a separate SLProtocol and SLScripting instance because of memory load.|
```